<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorForm</class>
 <widget class="QWidget" name="EditorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1154</width>
    <height>872</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
               color: black;
               padding: 4px;
               border: 1px solid rgb(63, 127, 191); 
               border-radius: 5px;
               border-style: outset;
              }
QWidget {
               color: rgb(32, 74, 135);
	           font: 11pt &quot;MS Shell Dlg 2&quot;;
              }
QScrollBar {
			   background-color: rgb(240, 240, 240);
              }
QLabel {
               color: rgb(32, 74, 135);
               padding: 4px;
               border-radius: 5px;
               border-style: outset;
	           font: 11pt &quot;MS Shell Dlg 2&quot;;
              }
QLineEdit {
               color: black;
               padding: 4px;
               border: 1px solid rgb(63, 127, 191); 
               border-radius: 5px;
               border-style: outset;
	           font: 10pt &quot;MS Shell Dlg 2&quot;;
              }
QComboBox {
			   color: black;
               padding: 4px;
               border: 1px solid rgb(63, 127, 191); 
               border-radius: 5px;
               border-style: outset;
               font: 9pt &quot;MS Shell Dlg 2&quot;;
			   background-color: rgb(255, 255, 255);
              }
QPlainTextEdit {
               color: black;
               padding: 4px;
               border: 1px solid rgb(63, 127, 191); 
               border-radius: 5px;
               border-style: outset;
	           font: 10pt &quot;MS Shell Dlg 2&quot;;
              }
QSpinBox {
               color: black;
               padding: 4px;
               border: 1px solid rgb(63, 127, 191); 
               border-radius: 5px;
               border-style: outset;
	           font: 10pt &quot;MS Shell Dlg 2&quot;;
              }

QToolButton:hover {
               color: white;
               border: 1px solid grey;
               background-color: rgb(114, 159, 207);
               }

QToolButton:pressed {
               color: rgb(0, 0, 197)
               background: shadow;
               border-style: inset;
               }</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Worker</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="workComboBox">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Equipment</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="equipComboBox">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Date</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="dateEdit">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QDateEdit {
               color: black;
               padding: 4px;
               border: 1px solid rgb(63, 127, 191); 
               border-radius: 5px;
               border-style: outset;
	           font: 10pt &quot;MS Shell Dlg 2&quot;;
              }</string>
            </property>
            <property name="dateTime">
             <datetime>
              <hour>0</hour>
              <minute>0</minute>
              <second>0</second>
              <year>2018</year>
              <month>7</month>
              <day>1</day>
             </datetime>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QToolButton" name="updateButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QToolButton {
                color: darkblue;
               border: 1px solid rgb(0, 85, 255); 
               border-radius: 5px;
               border-style: outset;
              }

QToolButton:hover {
               color: white;
               border: 1px solid grey;
               background-color: rgb(0, 78, 235);
               }

QToolButton:pressed {
               color: rgb(0, 0, 197)
               background: shadow;
               border-style: inset;
               }</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../res/res.qrc">
              <normaloff>:/refresh.png</normaloff>:/refresh.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="addButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QToolButton {
                color: darkblue;
               border: 1px solid rgb(0, 85, 255); 
               border-radius: 5px;
               border-style: outset;
              }

QToolButton:hover {
               color: white;
               border: 1px solid grey;
               background-color: rgb(0, 78, 235);
               }

QToolButton:pressed {
               color: rgb(0, 0, 197)
               background: shadow;
               border-style: inset;
               }</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../res/res.qrc">
              <normaloff>:/add.png</normaloff>:/add.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QTableView" name="tableView">
        <property name="styleSheet">
         <string notr="true">QTableView {
               color: black;
               border: 1px solid  rgb(63, 127, 191); 
               border-radius: 5px;
	           font: 11pt &quot;MS Shell Dlg 2&quot;;
              }
QHeaderView::section {
   			padding: 4px;
    		height: 40;
}</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QToolButton" name="languageButton">
             <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">QToolButton {
                color: darkblue;
               padding: 4px;
               border: 1px solid rgb(0, 85, 255); 
               border-radius: 5px;
               border-style: outset;
               font: 10pt &quot;MS Shell Dlg 2&quot;;
              }

QToolButton:hover {
               color: white;
               border: 1px solid grey;
               background-color: rgb(0, 78, 235);
               }

QToolButton:pressed {
               color: rgb(0, 0, 197)
               background: shadow;
               border-style: inset;
               }</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../res/res.qrc">
               <normaloff>:/english.png</normaloff>:/english.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="rejectButton">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QToolButton {
                color: darkblue;
               padding: 4px;
               border: 1px solid rgb(0, 85, 255); 
               border-radius: 5px;
               border-style: outset;
               font: 10pt &quot;MS Shell Dlg 2&quot;;
              }

QToolButton:hover {
               color: white;
               border: 1px solid grey;
               background-color: rgb(0, 78, 235);
               }

QToolButton:pressed {
               color: rgb(0, 0, 197)
               background: shadow;
               border-style: inset;
               }</string>
             </property>
             <property name="text">
              <string>Reject</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Minimum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>10</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="submitButton">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QToolButton {
               color: white;
               padding: 4px;
               border: 1px solid grey; 
               border-radius: 5px;
               border-style: outset;
               max-width: 120px;
               background-color: rgb(0, 85, 255);
               font: 10pt &quot;MS Shell Dlg 2&quot;;
              }

QToolButton:hover {
               color: darkblue;
               border: 1px solid grey;
               background-color: rgb(255, 255, 255);
               }

QToolButton:pressed {
               color: rgb(0, 0, 197)
               background: shadow;
               border-style: inset;
			    background-color: rgb(0, 78, 235);
               }</string>
             </property>
             <property name="text">
              <string>Submit</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionEnglish">
   <property name="icon">
    <iconset resource="../res/res.qrc">
     <normaloff>:/english.png</normaloff>:/english.png</iconset>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionRussian">
   <property name="icon">
    <iconset resource="../res/res.qrc">
     <normaloff>:/russian.png</normaloff>:/russian.png</iconset>
   </property>
   <property name="text">
    <string>Russian</string>
   </property>
  </action>
  <action name="actionUkrainian">
   <property name="icon">
    <iconset resource="../res/res.qrc">
     <normaloff>:/ukrainian.png</normaloff>:/ukrainian.png</iconset>
   </property>
   <property name="text">
    <string>Ukrainian</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res/res.qrc"/>
 </resources>
 <connections/>
</ui>
